aea/configurations/schemas/aea-config_schema.json

```diff
@@ -98,6 +98,21 @@
     },
     "description": {
       "$ref": "definitions.json#/definitions/description"
+    },
+    "timeout": {
+      "type": ["number", "null"],
+      "minimum": 0
+    },
+    "execution_timeout": {
+      "type": ["number", "null"],
+      "minimum": 0
+    },
+    "max_reactions": {
+      "type": ["integer", "null"],
+      "minimum": 1
+    },
+    "search_service_address": {
+    	"type": ["string", "null"]
     }
```

Comment:
please move all the type definitions to `definitions`

Reply:
done